src/main/java/com/vonage/quickstart/meetings/CreateLongTermRoom.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2023 Vonage
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package com.vonage.quickstart.meetings;
+
+import com.vonage.client.VonageClient;
+import com.vonage.client.meetings.MeetingRoom;
+import static com.vonage.quickstart.Util.configureLogging;
+import static com.vonage.quickstart.Util.envVar;
+
+public class CreateLongTermRoom {
+
+	public static void main(String[] args) throws Exception {
+		configureLogging();
+
+		String VONAGE_APPLICATION_ID = envVar("VONAGE_APPLICATION_ID");
+		String VONAGE_PRIVATE_KEY_PATH = envVar("VONAGE_PRIVATE_KEY_PATH");
+		String DISPLAY_NAME = envVar("DISPLAY_NAME");
+
+		VonageClient client = VonageClient.builder()
+				.applicationId(VONAGE_APPLICATION_ID)
+				.privateKeyPath(VONAGE_PRIVATE_KEY_PATH)
+				.build();
+
+		MeetingRoom room = MeetingRoom.builder(DISPLAY_NAME).build();
```

Comment:
I think you forgot to create this room as long-term, right?
A long-term room requires the type `long_term` and an expiration date.